src/PageContent/AssignVolunteers/AssignOption.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+
+class AssignOption extends Component {
+
+    constructor(props) {
+        super(props);
+        let date = new Date(this.props.delivery.date);
+        this.state = {
+            day: moment(date).format('dddd'),
+            date: moment(date).format('l'),
```

Comment:
`moment(date).format('MM/DD')` for what you want

Reply:
Fixed